src/users/routes.test.ts

```diff
@@ -58,8 +59,9 @@ describe(`Routes: ${ROUTE_NAME}`, () => {
 
     it('sets validation on request payload', () => {
       const payload = router.options.validate.payload;
-      expect(payload.firstname).toBeDefined();
```

Comment:
Please bring back these `.expect` calls; they are crucial. We need to guarantee that this upgrade is not breaking existing functionality.

Reply:
router.options.validate.payload  changed in  v20. Returns an AST to show the schema definition, hence properties  like firstName are not defined on the root object anymore.